provider/labels.go

```diff
@@ -15,18 +17,24 @@ import (
 // tfbridge.SchemaInfo.
 func fixLabelNames(prov *tfbridge.ProviderInfo) {
 	// Recursively applies the label fix
-	var apply func(string, shim.Schema, *tfbridge.SchemaInfo)
-	apply = func(key string, m shim.Schema, info *tfbridge.SchemaInfo) {
+	var apply func(string, shim.Schema, *tfbridge.SchemaInfo, *[]resource.PropertyPath, resource.PropertyPath)
+
```

Comment:
Style nit: I prefer to not have a line separation. This makes it clearer to me that we are declaring and then assigning to the same var.
```suggestion
	var apply func(string, shim.Schema, *tfbridge.SchemaInfo, *[]resource.PropertyPath, resource.PropertyPath)
```

Feel free to ignore me here if it's clearer for you.